en/developers-guide/data-availability-layer-api.md

```diff
@@ -0,0 +1,400 @@
+---
+description: Integrate pull based oracle through Data Availability Layer API
+---
+
+# Pull Oracle
+
+Through the pull-based oracle, price data can be updated on-chain on demand by the user. Off-chain price data, which is updated at subsecond intervals, can be accessed and submitted to the chain on demand.
+
+## API Endpoint
+
+- Testnet (REST): https://dal.baobab.orakl.network/api/v1
+- Mainnet (REST): https://dal.cypress.orakl.network/api/v1
+
+The API requires the `X-API-Key` header for both REST API calls and WebSocket connections. Please [contact us](mailto:business@orakl.network) to receive a valid API key.
```

Comment:
I wonder if the biz team would want to include some pricing plans in the docs...

Reply:
it is dev docs so I think it should be okay 🤔